File: src/main/java/com/github/joelws/release/tracker/dao/release/ReleaseDao.java
package com.github.joelws.release.tracker.dao.release;

import com.github.joelws.release.tracker.dao.GenericDao;
import com.github.joelws.release.tracker.entity.release.Release;

import java.util.List;

public interface ReleaseDao extends GenericDao<Release, String>
{
    List<Release> list();
}

File: src/main/kotlin/com/github/joelws/release/tracker/dao/release/ReleaseDao.kt
package com.github.joelws.release.tracker.dao.release

import com.github.joelws.release.tracker.dao.GenericDao
import com.github.joelws.release.tracker.entity.release.Release

interface ReleaseDao : GenericDao<Release, String> {
    fun list(): List<Release>
}
